### ISSUE TITLE
Backup All Accounts

### ISSUE THREAD

**Issue by JulianTosh** [bitpay/copay#6217]
I have over a dozen accounts on my phone. I would like the option to be able to backup and restore all accounts in my copay wallet in one single operation. 

This will make my life much easier when my phone gets lost/stolen/broken and for traveling internationally and I don't want any bitcoins on my phone during the customs process.

**Comment by shadab-entrepreneur** [comment]
What if we can associate the android version of copay with MMI code so that when the user dials that particular code then only that app will start else it will be hidden on the device so that no other person can find it even if he/she checks the phone/tablets?

**Comment by flrebroker** [comment]
Good point Julian.  I thought what you are describing as a feature request would be the default setting.  I realized this is not the case when I created wallet #2 on my device and then got the warning that I need to back this up.  So for every wallet you have to hide a different 12 word seed in multi-locations to be safe?!  I don't know how you have so many wallets on your phone!

**Comment by JulianTosh** [comment]
They are all brain wallets too! 

Basically... sha256("Wallet name/description" + "cosigner ID" + "strong and memorable password"), then this entropy is used to [generate my own recovery phrases](https://github.com/iancoleman/bip39) for any wallet/account I ever need. The password stays in my head and I keep a list of accounts out in the semi-open. It's a bit more complicated than that, but it works awesome!
